src/lib/arrays.js

```diff
@@ -0,0 +1,113 @@
+const getNthElement = (index, array) => {
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
+  return array[index];
+};
+
+const arrayToCSVString = array => {
+  return array.toString();
+};
+
+function csvStringToArray(string) {
+  return string.split(',', 5);
+}
+
+const addToArray = (elementX, arrayX) => {
+  arrayX.push(elementX);
+};
+
+const addToArray2 = (NEWelement, ORIGarray) => {
+  const addToArray3 = ORIGarray.concat(NEWelement);
+  return addToArray3;
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(function(A) {
+    return A.toUpperCase();
+  });
+};
+
+const reverseWordsInArray = strings => {
+  return strings.map(function(A) {
+    return A.split('')
+      .reverse()
+      .join('');
+  });
+};
+
+const onlyEven = numbers => {
+  // const numbers2 = numbers.filter(a => a % 2 === 0); works, replaced filter with map,but this gives T or F//
+  return numbers.filter(a => a % 2 === 0);
+};
+
+const removeNthElement2 = (index, array) => {
+  const newArray = [...array];
+  if (index >= 0) {
+    newArray.splice(index, 1);
+    return newArray;
+  }
+};
+
+const elementsStartingWithAVowel = strings => {
+  const results = [];
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < strings.length; i++) {
+    if (
+      strings[i].startsWith('a') ||
+      strings[i].startsWith('e') ||
+      strings[i].startsWith('i') ||
+      strings[i].startsWith('o') ||
+      strings[i].startsWith('u') ||
+      strings[i].startsWith('A') ||
+      strings[i].startsWith('E') ||
+      strings[i].startsWith('I') ||
+      strings[i].startsWith('O') ||
+      strings[i].startsWith('U')
+    ) {
+      results.push(strings[i]);
+    }
+  }
+  return results;
```

Comment:
The basic idea is here, but theres a few different ways to go about this. This has a lot of repetition, and isnt that maintainable so should be re-written. One way is to define the vowels in a variable, then use a filter to validate if the first letter contains something in that variable eg:

```
  return strings.filter(string => {
    const vowels = 'aeiou';
    return vowels.includes(string.toLowerCase()[0]);
  });
```
Much more concise and does the exact same job